### PR TITLE
Add Plutip builder tool

### PR DESCRIPTION
**Pull request by mikekeke**
* Name: *Plutip*
* Description: *Cardano tool that aims to help dApp developers with integration testing and contracts debugging using disposable private network*
* Link:  https://github.com/mlabs-haskell/plutip
* Source: https://github.com/mlabs-haskell/plutip
* Tags:
  * `educational`
  * `opensource`

### CODE REVIEW

File: src/data/builder-tools.js
Comment (rphair) [inline src/data/builder-tools.js:771]:
```suggestion
    tags: ["haskell", "testing"],
```
Using undefined tags doesn't create them automatically: they have to be deliberately added.  One way of resolving this would be to add a `privatenetwork` tag (our convention doesn't use spaces or hyphens) but that would be done by reviewers & by consensus.  Until if & when that happens the simplest thing is just to drop the tag.

**Review by rphair — DISMISSED** [review]
Looks like a wonderful tool 😍

File: src/data/builder-tools.js
Comment (fill-the-fill) [inline src/data/builder-tools.js:770]:
```suggestion
    getstarted: null,
```

There is a specific section in get started on developer portal dedicated to builder tools get started tutorials. The format is in markdown. Feel free to add your guide there as well, otherwise please write null in that field.

Comment (fill-the-fill) [inline src/data/builder-tools.js:770]:
```suggestion
    getstarted: null,
```

There is a specific section in get started on developer portal dedicated to builder tools get started tutorials. The format is in markdown. Feel free to add your guide there as well, otherwise please write null in that field.

**Review by fill-the-fill — CHANGES_REQUESTED** [review]

File: src/data/builder-tools.js
Comment (mikekeke) [reply src/data/builder-tools.js]: Will it be possible to add "getting started" later?
We are still stabilizing our API, so I put `null` for now.

File: src/data/builder-tools.js
Comment (mikekeke) [reply src/data/builder-tools.js]: Thank you for review.
Tag dropped.

File: src/data/builder-tools.js
Comment (rphair) [reply src/data/builder-tools.js]: yes, we can approve this entry now & you can submit another PR to add the `getstarted:` link whenever you're ready 🤓

Comment (mikekeke) [reply src/data/builder-tools.js]: Great!
I've put `null` there with latest commit.

Thank you for review!

**Review by rphair — APPROVED** [review]
Nonexistent tag & `getstarted:` link removed.
Verified it builds OK with `staging` as this moment. ✔️

**Comment by rphair** [comment]
apologies @fill-the-fill GitHub doesn't seem to leave an accurate record of the review history after selecting "Dismiss review" to clear the red-flag bits.  I'll try to do it more cleanly next time (I was looking for a way to officially mark that our requested changes had been made) 😖

**Review by fill-the-fill — APPROVED** [review]

**Comment by mikekeke** [comment]
Hey!
Is there any chance to get 3d approval from somebody to merge it?
cc @rphair @fill-the-fill

**Review by rdlrt — APPROVED** [review]
Current PR has a branch conflict due to the process. 

I am preemptively approving to avoid wait times in case another commit gets merged and this one has to be updated again

**Comment by rphair** [comment]
thanks @rdlrt - yes that's quite a few that came in in the meantime; moved Plutip to the end.  Also double checked it still builds after the commit to resolve the merge conflict ✔️

**Pull request merged**
